fastplotlib/layouts/_plot_area.py

```diff
@@ -123,6 +126,10 @@ def __init__(
         self.scene.add(self._ambient_light)
         self.scene.add(self._camera.add(self._directional_light))
 
+        self._tooltip = Tooltip()
+        self.get_figure()._fpl_overlay_scene.add(self._tooltip._fpl_world_object)
+        self.renderer.add_event_handler(self._fpl_set_tooltip, "pointer_move")
```

Comment:
`PlotArea` manages a `Tooltip` instance. Tooltips are draw in the overlay render pass, and are therefore in screen space. The `"pointer_move"` event determines where it appears (in screen space). Therefore there is only ever 1 tooltip instance (and thus only one mesh, text, and line object required to draw it) per `PlotArea`. If they're managed by a graphic then things get complicated.